lib/utils.pm

```diff
@@ -1221,4 +1223,25 @@ sub script_run_interactive {
     }
 }
 
+sub setup_autofs_server {
+    my ($autofs_conf_file, $autofs_map_file, $test_conf_file, $test_conf_file_content, $test_mount_dir) = @_;
+    my $grep_output = script_output("grep '#+dir' $autofs_conf_file");
+    if ($grep_output =~ /\#\+dir/){
+        assert_script_run("sed -i_bk 's:#+dir\\:/etc/auto\\.master\\.d:+dir\\:/etc/auto\\.master\\.d:' $autofs_conf_file");
+    }
+    assert_script_run("echo $test_mount_dir $test_conf_file > $autofs_map_file");
+    assert_script_run($test_conf_file_content, timeout => 10, fail_message => "File $test_conf_file could not be created");
```

Comment:
Hm, do you have a specific reason to limit the timeout to less-than-default? I would just keep the default here.

Reply:
fixed ;)